src/bundler.ts

```diff
@@ -393,7 +393,7 @@ export class Bundler {
       ast: ASTNode,
       bundle: AssignedBundle,
       bundleManifest: BundleManifest) {
-    const visitedUrls = new Set<UrlString>();
+    const visitedUrls = new Set<UrlString>(bundle.bundle.stripImports);
```

Comment:
`visitedUrls` isn't really accurate anymore, since it's pre-populated with URLs that weren't actually visited (but you just want the side effect that "visited" currently conveys).

Not a huge deal, but could be more clear